src/App.test.js

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import App from './App';
+jest.mock('./components/Header', () => 'Header');
+jest.mock('react-router-dom/Route', () => 'Route');
 
-it('renders without crashing', () => {
-  shallow(<App />);
+describe('<App />', () => {
+  const wrapper = shallow(<App />);
+  const routes = [
+    {
+      path: '/',
+      component: 'Home'
+    },
+    {
+      path: '/shelters',
+      component: 'Shelters'
+    },
+    {
+      path: '/hygiene',
+      components: 'Hygiene'
+    },
+    {
+      path: '/hotlines',
+      component: 'Hotlines'
+    },
+    {
+      path: '/food',
+      component: 'Food'
+    },
+    {
+      path: '/transit',
+      component: 'Transit'
```

Comment:
good call on keeping the components as strings since we don't care about the exact components.